variables.tf

```diff
@@ -484,6 +484,32 @@ variable "bitbucket_elasticsearch_replicas" {
   default     = 2
 }
 
+variable "bitbucket_db_snapshot_identifier" {
+  description = "The identifier for the DB snapshot to restore from. The snapshot should be in the same AWS region as the DB instance."
+  default     = null
+  type        = string
+}
+
+variable "bitbucket_db_master_username" {
+  description = "Master username for the Bitbucket RDS instance."
+  type        = string
+  default     = null
+  validation {
+    condition     = can(regex("^[a-zA-Z_]([a-zA-Z0-9_]).{5,30}$", var.bitbucket_db_master_username)) || var.bitbucket_db_master_username == null
```

Comment:
Is it Bitbucket specific or postgres specific requirement?

Reply:
I did a search on postgress requirement, but now I double checked with AWS RDS and it looks this should be adjust with the following: `^[a-zA-Z]([a-zA-Z0-9]?).{0,14}$`

From Create Database page in AWS console:

> Master Username:  1 to 16 alphanumeric characters. First character must be a letter.